src/highdicom/sc/sop.py

```diff
@@ -125,43 +126,43 @@ def __init__(
             Name of the manufacturer of the device that creates the SC image
             instance (in a research setting this is typically the same
             as `institution_name`)
-        patient_id: str, optional
+        patient_id: Union[str, None], optional
            ID of the patient (medical record number)
-        patient_name: Optional[Union[str, PersonName]], optional
+        patient_name: Union[str, PersonName, None], optional
            Name of the patient
-        patient_birth_date: str, optional
+        patient_birth_date: Union[str, None], optional
            Patient's birth date
-        patient_sex: str, optional
+        patient_sex: Union[str, highdicom.enum.PatientSexValues, None], optional
            Patient's sex
-        study_id: str, optional
+        study_id: Union[str, None], optional
            ID of the study
-        accession_number: str, optional
+        accession_number: Union[str, None], optional
            Accession number of the study
-        study_date: Union[str, datetime.date], optional
+        study_date: Union[str, datetime.date, None], optional
            Date of study creation
-        study_time: Union[str, datetime.time], optional
+        study_time: Union[str, datetime.time, None], optional
            Time of study creation
-        referring_physician_name: Optional[Union[str, PersonName]], optional
+        referring_physician_name: Union[str, PersonName, None], optional
```

Comment:
```suggestion
        referring_physician_name: Union[str, pydicom.valuerep.PersonName, None], optional
```

Reply:
I just realized that we forgot to expose the content of the `enum` module at the package level. I would suggest exposing the enum classes similar to those in the sub-packages (`seg`, `sr`). What do you think?

Reply:
Resolved in 9a6f15f

Reply:
Exposed enum at the root as suggested in 9a6f15f